### ISSUE TITLE
Gradle tests are failing

### ISSUE THREAD

**Issue by reichhartd** [AppsFlyerSDK/appsflyer-react-native-plugin#306]
# Report

## Plugin Version

6.3.20

## On what Platform are you having the issue?

Android

## What did you do?

Run `cd android && ./gradlew test`

## What did you expect to happen?

That the gradle tests pass

## What happened instead?

They are terminated with an error

## Please provide any other relevant information.

Logs:

```sh
2021-07-01T17:15:31.3228858Z > Task :react-native-appsflyer:compileDebugUnitTestJavaWithJavac FAILED
2021-07-01T17:15:31.3233203Z /home/gharunner/runner6/gha-runner1-6/app/app/node_modules/react-native-appsflyer/android/src/test/java/PluginNativeTests.java:44: error: <anonymous PluginNativeTests$1> is not abstract and does not override abstract method reject(String,String,Throwable,WritableMap) in Promise
2021-07-01T17:15:31.3235757Z     private Promise successPromise = new Promise() {
2021-07-01T17:15:31.3236542Z                                                    ^
2021-07-01T17:15:31.3239582Z /home/gharunner/runner6/gha-runner1-6/app/app/node_modules/react-native-appsflyer/android/src/test/java/MockMap.java:12: error: MockMap is not abstract and does not override abstract method toHashMap() in ReadableMap
2021-07-01T17:15:31.3241618Z public class MockMap implements ReadableMap {
2021-07-01T17:15:31.3242236Z        ^
2021-07-01T17:15:31.3245995Z /home/gharunner/runner6/gha-runner1-6/app/app/node_modules/react-native-appsflyer/android/src/test/java/AppsFlyerLibMock.java:17: error: AppsFlyerLibMock is not abstract and does not override abstract method setDisableAdvertisingIdentifiers(boolean) in AppsFlyerLib
2021-07-01T17:15:31.3248628Z public class AppsFlyerLibMock extends AppsFlyerLib {
2021-07-01T17:15:31.3249323Z        ^
2021-07-01T17:15:31.3250127Z Note: Some input files use or override a deprecated API.
2021-07-01T17:15:31.3251396Z Note: Recompile with -Xlint:deprecation for details.
2021-07-01T17:15:31.3252082Z 3 errors
2021-07-01T17:15:31.3252391Z 
2021-07-01T17:15:31.3253091Z > Task :app:mergeProdDebugResources
2021-07-01T17:15:31.3253604Z 
2021-07-01T17:15:31.3254287Z FAILURE: Build failed with an exception.
2021-07-01T17:15:31.3254727Z 
2021-07-01T17:15:31.3255231Z * What went wrong:
2021-07-01T17:15:31.3256837Z Execution failed for task ':react-native-appsflyer:compileDebugUnitTestJavaWithJavac'.
2021-07-01T17:15:31.3258290Z > Compilation failed; see the compiler error output for details.
2021-07-01T17:15:31.3258986Z 
2021-07-01T17:15:31.3259411Z * Try:
2021-07-01T17:15:31.3261115Z Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
2021-07-01T17:15:31.3261901Z 
2021-07-01T17:15:31.3262683Z * Get more help at https://help.gradle.org
2021-07-01T17:15:31.3263200Z 
2021-07-01T17:15:31.3264257Z Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
2021-07-01T17:15:31.3265800Z Use '--warning-mode all' to show the individual deprecation warnings.
2021-07-01T17:15:31.3267439Z See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings
2021-07-01T17:15:31.3268225Z 
2021-07-01T17:15:31.3268692Z BUILD FAILED in 13m 17s
2021-07-01T17:15:31.3269397Z 784 actionable tasks: 784 executed
```

**Comment by amit-kremer93** [comment]
Hi @reichhartd and thank you for opening this issue.
We will fix this issue in the next release, but for now you can simply remove the `node_modules/react-native-appsflyer/android/src/test` folder.

**Comment by amit-kremer93** [comment]
Fixed on 6.3.50
